.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   test:
-    if: github.event.pull_request.draft == false
+    # if: github.event.pull_request.draft == false
```

Comment:
Yes the correct change is to move this line, but it should be done in a separate pull request since it's unrelated to the change you're working on.